examples/example_tf_transform/taxi.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import tensorflow as tf
+
+
+def transformed_name(key):
+    return key + "_xf"
+
+
+def fill_in_missing(x):
+    default_value = "" if x.dtype == tf.string else 0
```

Comment:
Are there other types which require defaults? Or are string and int the only two?

Reply:
string/float/int, 0 also works for floats